src/main/java/uk/gov/hmcts/dts/fact/model/Court.java

```diff
@@ -53,43 +56,45 @@
     private List<CourtAddress> addresses;
     private String gbs;
     @JsonProperty("dx_number")
-    private List<String> dxNumber;
+    private List<String> dxNumbers;
     @JsonProperty("service_area")
     private String serviceArea;
     @JsonProperty("in_person")
     private Boolean inPerson;
 
     public Court(uk.gov.hmcts.dts.fact.entity.Court courtEntity) {
-        this.name = courtEntity.getName();
+        this.name = chooseString(courtEntity.getNameCy(), courtEntity.getName());
         this.slug = courtEntity.getSlug();
-        this.info = Filters.stripHtmlFromString(courtEntity.getInfo());
+        this.info = stripHtmlFromString(chooseString(courtEntity.getInfoCy(), courtEntity.getInfo()));
         this.open = courtEntity.getDisplayed();
-        this.directions = courtEntity.getDirections();
+        this.directions = chooseString(courtEntity.getDirectionsCy(), courtEntity.getDirections());
```

Comment:
didnt know direction was different for welsh language? thought it was just a number.

Reply:
Cardiff Crown Court has "The court is situated next to the City Hall and Central Police Station" and "Mae'r Llys wedi ei leoli nesaf at Neuadd y Ddinas a Gorsaf Heddlu Ganolog" for English and Welsh respectively